### PR TITLE
 * Un-pin dotnet version

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Make KoreBuild build with the OS specific versions saved for cli.version.\* files.

### CODE REVIEW

**Comment by victorhurdugaci** [comment]
Can we get this merged ASAP? Just create the 3 files as part of this PR

File: build/KoreBuild.ps1 [hunk 1]
```diff
@@ -11,7 +11,7 @@ $koreBuildFolder = $PSScriptRoot
 $koreBuildFolder = $koreBuildFolder.Replace($repoFolder, "").TrimStart("\")
 
 $dotnetChannel = "beta"
-$dotnetVersion = "1.0.0.001540"
+$dotnetVersion = [IO.File]::ReadAllText(".\cli.version.win")
```
Comment (natemcmaster) [inline build/KoreBuild.ps1]: Use ps command instead? `Get-Content .\cli.version.win`

**Comment by ryanbrandenburg** [comment]
I guess that's a :shipit: then?

File: build/cli.version.darwin [hunk 2]
```diff
@@ -0,0 +1 @@
+1.0.0.001540
```
Comment (natemcmaster) [inline build/cli.version.darwin:1]: ~~Double check this. I believe we want a different version on osx... likewise for linux.~~

Comment (natemcmaster) [inline build/cli.version.darwin:1]: nvm. Scratch that. I'm working off old data.

Comment (ryanbrandenburg) [inline build/cli.version.darwin:1]: This is the version it had before, was that wrong? I'm not trying to give it something up to date, just get it ready for the CoreCLR changes.

Comment (ryanbrandenburg) [inline build/cli.version.darwin:1]: Ah didn't see your update. k.

**Comment by ryanbrandenburg** [comment]
I'm going to merge this first thing in the morning.

**Comment by victorhurdugaci** [comment]
:shipit:

**Pull request merged**